<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTutorialDlg</class>
 <widget class="QWidget" name="CTutorialDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>330</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg&quot;, 0, 0, 0x</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Application Update GUI Elements</string>
  </property>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>11</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Click to Add</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>292</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_IDC_STATIC">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>18</y>
     <width>75</width>
     <height>13</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Button Clicks:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="echoArea">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>16</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="vSliderBar">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>31</y>
     <width>25</width>
     <height>232</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hSliderBar">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>293</y>
     <width>243</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Label_IDC_STATIC1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>58</width>
     <height>13</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Vertical Bar</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Label_IDC_STATIC2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>268</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Horizontal Bar</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="vSliderEcho">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>16</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Static</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hSliderEcho">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>268</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Static</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="timerecho">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>234</y>
     <width>219</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mouseecho">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>211</y>
     <width>219</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Label_IDC_STATIC3">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>235</y>
     <width>67</width>
     <height>13</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Timer Echo:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Label_IDC_STATIC4">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>214</y>
     <width>67</width>
     <height>13</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Mouse Echo:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="timerControlSliders">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>122</y>
     <width>129</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Timer Control Sliders</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
